app/src/main/java/com/example/p1/ui/theme/WelcomePage.kt

```diff
@@ -0,0 +1,145 @@
+package com.example.ttrip.ui.theme
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.blur
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shadow
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.p1.R
+
+
+@Composable
+fun WelcomePage() {
+    val painter = painterResource(id = R.drawable.intro)
+    val description = "first page"
+
+    imageA(painter = painter, contentDescription = description)
+}
+
+@Composable
+fun imageA(
+    painter: Painter,
```

Comment:
1-Try to name your composable with descriptive name.
2-Composable functions should be named with capital letter not imageA ->ImageA 
3-if a composable (or any function) is used only in one place you should make it private (Read about access modifier in any language(public, private..))